support/webpack.config.js

```diff
@@ -45,15 +45,19 @@ var config = {
           {
             loader: "eslint-loader",
             options: {
-              emitError: true,
-              emitWarning: true,
-              failOnWarning: false,
-              failOnError: false
+              // emitError: true,
```

Comment:
I think these rules are too strict for development. For example, adding a console.log temporarily to try to figure out a bug prevents the webpack from running so you can't test at all. I can see having these for a production build, but otherwise I'd rather let Atom tell me there's an error than have the build process stop.